docs/docs/reference/reference/smart_contract_reference/storage/index.md

```diff
@@ -22,22 +22,32 @@ On this and the following pages in this section, you’ll learn:
 - Practical implications of Storage in real smart contracts
   In an Aztec.nr contract, storage is to be defined as a single struct, that contains both public and private state variables.
 
-## Public and private state variables
```

Comment:
This section didn't really seem to add anything here: the concepts are barely mentioned, whereas we have both high and low level explanations in other places. I think we can just remove this part and rely on the other sections to cover this in detail. I added links to help users find those.